package.json

```diff
@@ -36,6 +36,7 @@
     "stripe-angular": "0.5.0",
     "swagger-ui": "2.2.10",
     "tslib": "1.10.0",
+    "yarn": "^1.19.1",
```

Comment:
This is unnecessary. The generator of this project auto installs yarn locally. If you go to your root folder of the project you'll see a npm directory and within it a yarn directory. This is where yarn is installed. :smile:

Reply:
yarn suggested to do 'yarn audit autofix' to fix some yarn irregularities. I did that and yarn has added this.